js/ui/components/clubhouse.js

```diff
@@ -291,6 +316,13 @@ var ClubhouseComponent = new Lang.Class({
             });
         }
 
+        if (this._clubhouseProxySuggestOpenHandler == 0) {
+            this._clubhouseProxySuggestOpenHandler = this.proxy.connectSignal('SuggestOpen', () => {
+                if (!Main.overview.visible)
```

Comment:
I don't think this check here is needed. `setOpenButtonHighlight()` does not affect the visibility of the button, which is what we care about the overview visibility for.